### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by suryavaddiraju**
included constants and utility directories destinations in setup file configuration file

relates to #223

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
     license="MIT",
     install_requires=["requests"],
     include_package_data=True,
-    package_dir={'razorpay': 'razorpay', 'razorpay.resources': 'razorpay/resources'},
-    packages=['razorpay', 'razorpay.resources'],
+    package_dir={'razorpay': 'razorpay', 'razorpay.resources': 'razorpay/resources', 'razorpay.constants': 'razorpay/constants', 'razorpay.utility': 'razorpay/utility'},
+    packages=['razorpay', 'razorpay.resources', 'razorpay/constants', 'razorpay/utility'],
```
Comment (kkyusufk) [inline setup.py:18]: Can you elaborate why these should be added here?

Comment (suryavaddiraju) [reply setup.py]: We have to show what directories are included with razorpay-python-master.zip package in setup.py file to say that the installing folders belongs to razorpay and they come with package
---------------------
 constants and utility folders of razorpay folder is not entered in setup.py file package directory  and packages line but they are in razorpay-python-master.zip package, so the warning attached in image appears
---------------------
**before this edit** during `python setup.py install` , check the warning
Screenshot (7)
**after this edit** during `python setup.py install` , the warning disappears
Screenshot (10)
----------------------

Comment (suryavaddiraju) [reply setup.py]: > Can you elaborate why these should be added here?

small change in 18th line and I have updated it, check it
placed `razorpay/constants` in the place of `razorpay.constants` and `razorpay/utility` in the place of `razorpay.utility`
the previously entered line is `packages=['razorpay', 'razorpay.resources', 'razorpay/constants', 'razorpay/utility'],` and corrected 18th line (packages definition line) is
`packages=['razorpay', 'razorpay.resources', 'razorpay.constants', 'razorpay.utility'],`

Comment (kkyusufk) [reply setup.py]: Thanks! Will review it and merge it soon!
Thank you for the contribution! 😄

Comment (suryavaddiraju) [reply setup.py]: thank you

**Pull request closed without merging**